### PR TITLE
Miscellaneous Virtualbox fixes

### PR DESCRIPTION
**Pull request by Kensan**
A couple of minor Virtualbox-related fixes.

### CODE REVIEW

File: repos/ports/run/vbox_win.inc [hunk 1]
```diff
@@ -74,7 +74,7 @@ append config_of_app {
 		needs to be enabled in the .vbox file (disabled by default).
 		(Recording is configured to use the external mic.)
 	-->
-	<start name="audio_drv">
+	<start name="audio_drv" priority="-1">
```
Comment (chelmuth) [inline repos/ports/run/vbox_win.inc]: I'm surprised that a priority downgrade of the audio driver eliminates choppy sound. Did you investigate the interactions of the scenario?

**Comment by chelmuth** [comment]
I merged the PR but skipped 9e413148b520f61ed0be05da60c580d304a89f78 for now.

File: repos/ports/run/vbox_win.inc (hunk 1 above)
Comment (Kensan) [inline repos/ports/run/vbox_win.inc]: The choppyness is not eliminated completely but the quality of the output is improved. The issue was discovered when testing audio on hw_x86_64_muen/Virtualbox and @cnuke suggested the change until he has time to investigate the underlying cause. Maybe @cnuke can comment if it still should be applied.

**Comment by cnuke** [comment]
9e41314 should be applied because all other drivers are also running on priority `-1`and it was an oversight on my side that the audio_drv is not. There is no reason that it runs on the highest priority and in fact it also makes the sound choppy when running on NOVA. I still have not investigated why changing the priority of the driver influences the playback quality.

**Comment by chelmuth** [comment]
Thanks, 9e413148b520f61ed0be05da60c580d304a89f78 was merged just now.

**Comment by chelmuth** [comment]
Merged into master.

**Pull request closed without merging**